Templates/project_collection/README.md

```diff
@@ -0,0 +1,212 @@
+  * This is the common name of the species that this data collection is about
+  * This field is controlled by the vocabulary in the `species` column on `values` sheet
+  * You can submit multiple values for this field, separated by commas or 
+    new rows immediately after this one
+  * Usually there is only one species associated with a data collection
+  * Data collections about human subjects should be separate from other species
+  
+#### **is data of type** - *modality*
+  * This is the name of the data modality for this collection
+  * This field is controlled by the vocabulary in the `modality` column on `values` sheet
+  * You can submit multiple values for this field, separated by commas or 
+    new rows immediately after this one
+  * Usually there is only one modality associated with a data collection
+  
+#### **is subject of resource** - *URL*
+  * This is usually a link to the repository where the raw data is stored.
+  * You can submit multiple values for this field, separated by commas or 
+    new rows immediately after this one
+
+#### **was collected using** - *technique*
+  * This is the name technique used to acquire data for this collection
+  * This field is controlled by the vocabulary in the `technique` column on `values` sheet
+  * You can submit multiple values for this field, separated by commas or 
+    new rows immediately after this one
+  * Usually there is only one technique associated with a data collection
+  
+#### **Title** - *string*
+  * This is a simple string is currently not displayed. 
+    (note: Title _is_ displayed for _data collection projects_)
+    
+#### **Short title** - *string*
+  * This is a simple string that will be displayed on 
+    https://knowledge.brain-map.org/ as your data collection title
+    (note: Short title _is not_ displayed for _data collection projects_)
+  * Please limit short titles to 30 characters or less
+  
+#### **Description** - *string*
+  * This is a simple string that will be displayed on 
+    https://knowledge.brain-map.org/ as your data collection description
+  
+#### **Access control** - *access control*
+  * This indicates the level of control of access to these data
+  * This field is controlled by the vocabulary in the `access control` column on `values` sheet
+  * Most non-human data collections should be `open`
+  * Human data collections should be `controlled`
+
+#### **Completion state** - *completion state*
+  * This indicates whether or not new data is still being added to this collection
+  * This field is controlled by the vocabulary in the `completion state` column on `values` sheet
+  * If more data are expected to be added to this collection, then the state is `in progress`
+  * If data not more data will be added to this collection, then the state is `complete`
+  
+#### **Last updated date** - *yyyy/mm/dd*
+  * This is the day of the most recent data submission deadline where the `Reported count` was updated
```

Comment:
I think we should remove this from the template, and plan that this is an internal process for calculated when *we* last updated the dataset.

Reply:
Note this is a comment for future, does not need to be resolved now.

Reply:
Based on this and your feedback about specimen count, I think we should probably remove both fields from the template.

Reply:
Deleted from template and README